topdown/trace.go

```diff
@@ -231,15 +236,21 @@ func (b *BufferTracer) Config() TraceConfig {
 
 // PrettyTrace pretty prints the trace to the writer.
 func PrettyTrace(w io.Writer, trace []*Event) {
-	depths := depths{}
-	for _, event := range trace {
-		depth := depths.GetOrSet(event.QueryID, event.ParentID)
-		fmt.Fprintln(w, formatEvent(event, depth))
-	}
+	PrettyTraceWith(PrettyTraceOpts{}, w, trace)
```

Comment:
[nit] Let's make the options the last argument, I think that's more consistent with the rest of the code base.